gamestonk_terminal/etf/ETF_Config.ini

```diff
@@ -0,0 +1,23 @@
+[Price]
```

Comment:
Can this filename use lowercase? That's a PEP8 convention I believe

Reply:
so just etc_config.ini?

Reply:
Yup